src/lib/openapi/util/openapi-tags.ts

```diff
@@ -33,6 +33,11 @@ const OPENAPI_TAGS = [
             'Revive or permanently delete [archived feature toggles](https://docs.getunleash.io/advanced/archived_toggles).',
     },
     { name: 'Auth', description: 'Manage logins, passwords, etc.' },
+    {
+        name: 'Change Requests',
+        description:
+            'Operations related to [Change Requests](https://docs.getunleash.io/reference/change-requests).',
+    },
```

Comment:
This tag already existed https://github.com/Unleash/unleash/pull/4432/files#diff-3085a6dcb1b86d8668930f740073efd90f33357076947ad1da0307b856437f2aL122-L126 and is causing some issues generating the clients

Reply:
Oh dang, they already existed? I'll look into it.